src/legacy/ui/public/courier/fetch/components/_shard_failure_modal.scss

```diff
@@ -0,0 +1,50 @@
+// set width and height to fixed values to prevent resizing when you switch tabs
+.shardFailureModal {
+     height: 75vh;
+     width: 768px;
+}
+
+// displaying a EuiCodeBlock inside a tab inside a modal, you've to set the width 
+// to prevent overlapping the modal when the content contains very long lines
+.shardFailureModal__codeblock {
+    width: 702px
```

Comment:
This number feels odd. I'd just round it out to `700px`. But you should check out this works when the modal shrinks based on the browser width.

Reply:
thx, done. for the breakpoints `xs` , `s`, this is set to `auto`, otherwise the doesn't shrink

Reply:
note about 702px, it's the inner space available of the modal:

![Bildschirmfoto 2019-07-25 um 08 56 20](https://user-images.githubusercontent.com/463851/61852602-1fab9780-aeba-11e9-87d2-386d74381b86.png)

better solution would be, not so set it, but in this case it's displayed this way:
![Bildschirmfoto 2019-07-25 um 08 56 31](https://user-images.githubusercontent.com/463851/61852662-4bc71880-aeba-11e9-9e05-57b668104ef8.png)

Reply:
Looks like what your actual problem is is that long strings aren't wrapping causing the width of the container to expand. We allow the code blocks to scroll horizontally, so you might be able to just set `max-width: 100%` so that it doesn't expand out.

Reply:
So finally being able to see this modal, it looks like the problem stems from EUI. We need to hide overflow on the `.euiModal__flex` element:

<img src="https://d.pr/free/v/l0r1A1+" />

Reply:
```
.shardFailureModal .euiModal__flex {
    max-width: 100%;
}
```
would fix this, but we shouldn't use these EUI classes, right?

Reply:
I've got 2 PR's in EUI for you that will fix some of your issues:

1. https://github.com/elastic/eui/pull/2164 Will fix the modal. So just go ahead and remove your SASS for this and when EUI updates in Kibana, it will automatically get fixed.
2. https://github.com/elastic/eui/pull/2166 Will give you the option for "responsive" column description lists. But you can leave your SASS in for that right now since it will also take adding another prop later.

Reply:
@cchaos should I add an issue for this in the EUI in repo, leave it that way for now? thx!

Reply:
@kertal See response above

Reply:
seems like we wrote our comments at the same time :) awesome! thank you!